samples/HostConfig/sample.json

```diff
@@ -273,7 +273,6 @@
 	{
 		"label": 
 		{
-			"inputSpacing": "small",
```

Comment:
why is the removals of inputSpacing here and in `source/uwp/Visualizer/HostConfigs/DefaultHostConfig.json` being done?

Reply:
Removed these changes.

---
In reply to: [476877388](https://github.com/microsoft/AdaptiveCards/pull/4581#discussion_r476877388) [](ancestors = 476877388)